src/Featurizers/TfidfVectorizerFeaturizer.cpp

```diff
@@ -201,13 +201,16 @@ void TfidfVectorizerTransformer::execute_impl(typename BaseType::InputType const
             results.emplace_back(std::make_tuple(labelIter->second, tfidf));
         }
     }
-    //normVal will never be 0 as long as results is not empty
-    assert(normVal > 0.0f);
+    //normVal will be zero when the input is empty
+    assert(normVal >= 0.0f);

```

Comment:
>assert(normVal >= 0.0f); [](start = 4, length = 24)

I think that the logic is this:

the normValue will never be 0 if the NormMethod is L1 or L2. It will always be 0 if the NormMethod is None. 

Can you refactor this assert to make that explicit?

Reply:
I realized yesterday that when the input is empty, the normValue is 0